go/test/endtoend/utils/mysql.go

```diff
@@ -207,6 +207,10 @@ func compareVitessAndMySQLResults(t *testing.T, query string, vtQr, mysqlQr *sql
 	for _, row := range mysqlQr.Rows {
 		errStr += fmt.Sprintf("%s\n", row)
 	}
+	if vtConn != nil {
+		qr := Exec(t, vtConn, fmt.Sprintf("vexplain plan %s", query))
+		errStr += fmt.Sprintf("query plan: \n%s\n", qr.Rows[0][0].ToString())
+	}
```

Comment:
Is this for debugging or something you want to actually merge?

Reply:
this is a good change that will help. today we only show the plans if the difference is between v3 and gen4, but not when the difference is between mysql and vitess. this change helps us see the plans in the latter case as well